src/sentry/api/endpoints/api_authorizations.py

```diff
@@ -47,10 +48,11 @@ def delete(self, request: Request) -> Response:
         except ApiAuthorization.DoesNotExist:
             return Response(status=404)
 
-        with transaction.atomic(using=router.db_for_write(ApiToken)):
-            ApiToken.objects.filter(
+        with outbox_context(transaction.atomic(using=router.db_for_write(ApiToken)), flush=False):
```

Comment:
Now that ApiTokens are replicated, we must ensure that these operations generate outboxes correctly.